xml/System.Threading/Interlocked.xml

```diff
@@ -2221,7 +2221,7 @@ This method wraps a call to [FlushProcessWriteBuffers](/windows/win32/api/proces
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Threading.Interlocked.Read%2A> method is unnecessary on 64-bit systems, because 64-bit read operations are already atomic. On 32-bit systems, 64-bit read operations are not atomic unless performed using <xref:System.Threading.Interlocked.Read%2A>.  
+ The <xref:System.Threading.Interlocked.Read%2A> method is necessary on 64-bit systems when the location refers to a 64-bit value that may span across cachlines. On 32-bit systems, 64-bit read operations are not atomic unless performed using <xref:System.Threading.Interlocked.Read%2A>.  
```

Comment:
```suggestion
 The <xref:System.Threading.Interlocked.Read%2A> method is only necessary on 64-bit systems when the location refers to a 64-bit value that may span across cache lines. On 32-bit systems, 64-bit read operations are not atomic unless performed using <xref:System.Threading.Interlocked.Read%2A>.  
```

Reply:
Maybe there should also be a sentence about when the 64-bit value may span across cache line.
Or the other way around: Fields layed out by the CLI are aligned to 8 byte-boundaries, so this won't happen. Only structs with explicit layout (or there any other cases expect _unsafe_ code?) fall under this.
Right now I don't have any proper wording in mind for this.

Let's think of a ordinary C# developer which has first contacts with Interlocked, so there might be more confusion after reading the docs than before.
This topic isn't an easy one and the docs can't cover all aspect, but they should provide good enough guidance to get things done properly.